tomviz/modules/ModuleManager.cxx

```diff
@@ -948,21 +948,31 @@ void ModuleManager::onPVStateLoaded(vtkPVXMLElement*,
       QStringList fileNames;
       if (dsObject.contains("reader")) {
         auto reader = dsObject["reader"].toObject();
-        options["reader"] = reader;
 
         if (reader.contains("fileNames")) {
           foreach (const QJsonValue& value, reader["fileNames"].toArray()) {
             // Verify the file exists before adding it to the list.
             if (QFileInfo::exists(value.toString())) {
               fileNames << value.toString();
             } else {
-              qCritical() << "File" << value.toString() 
-                          << "not found, skipping.";
+              // If the file cannot be found in the path relative to the state
+              // file
```

Comment:
Try to fix the odd wrapping here to make it flow better (guessing clang-format did this).